src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/DataprocConfig.scala

```diff
@@ -1,11 +1,11 @@
 package org.broadinstitute.dsde.workbench.leonardo.config
 
-import org.broadinstitute.dsde.workbench.leonardo.model.GoogleServiceAccount
+import org.broadinstitute.dsde.workbench.model.WorkbenchEmail
 import org.broadinstitute.dsde.workbench.model.google.GoogleProject
 
 case class DataprocConfig(
                            applicationName: String,
-                           serviceAccount: GoogleServiceAccount,
+                           serviceAccount: WorkbenchEmail,
```

Comment:
Could we name this `serviceAccountEmail` since it's just an email? Not to be confused with [ServiceAccount](https://github.com/broadinstitute/workbench-libs/blob/d6801ce24a764c4e62cd00c3322b4d130bee6b4a/model/src/main/scala/org/broadinstitute/dsde/workbench/model/google/GoogleModel.scala#L11) which contains a subject id/display name as well.

Reply:
👍